src/components/text/text.styles.ts

```diff
@@ -0,0 +1,339 @@
+    }
+  
+    blockquote {
+      font-size: ${fontSize};
+      padding: ${lineHeightSize};
+    }
+
+    dd + dt {
+      ${logicalCSS('margin-top', fontSize)}
+    }
+
+    dt,
+    .eui-definitionListReverse dd {
+      font-size: ${fontSize};
+      line-height: ${lineHeight};
+    }
+
+    .eui-definitionListReverse dt {
+      font-size: ${euiFontSize('xs', euiTheme, options).fontSize};
+      color: ${euiTheme.colors.text};
+    }
+
+    small {
+      font-size: ${euiFontSize('s', euiTheme, options).fontSize};
+    }
+
+    pre {
+      padding: ${fontSize};
+    }
+
+    code {
+      font-size: .9em; // 90% of parent font size
+    }
+  `;
+};
+
+/**
+ * Styles
+ */
+export const euiTextStyles = ({ euiTheme }: UseEuiTheme) => ({
+  euiText: css`
+    ${euiText(euiTheme, true)};
+
+    // EuiImage with floats are often used within EuiText.
+    clear: both;
+
+    // Style anchors that don't have a class. This prevents overwriting "buttons"
+    // and other stylized elements passed in.
+    a:not([class]) {
+      ${euiLinkCSS(euiTheme)}
+    }
+
+    img {
+      display: block;
+      width: 100%;
+    }
+
+    ul {
+      list-style: disc;
+    }
+
+    ol {
+      list-style: decimal;
+    }
+
+    blockquote:not(.euiMarkdownFormat__blockquote) {
+      position: relative;
+      text-align: center;
+      margin-left: auto;
+      margin-right: auto;
+      font-family: Georgia, Times, Times New Roman, serif;
+      font-style: italic;
+      letter-spacing: normal;
+
+      p:last-child {
+        margin-bottom: 0;
+      }
+
+      &:before,
+      &:after {
+        position: absolute;
+        content: '';
+        height: 2px;
+        width: 50%;
+        right: 0;
+        transform: translateX(-50%);
+        background: ${euiTheme.colors.darkShade};
+      }
+
+      &:before {
+        top: 0;
+      }
+
+      &:after {
+        bottom: 0;
+      }
+    }
+
+    h1 {
+      ${euiTitle('l', euiTheme)}
+    }
+
+    h2 {
+      ${euiTitle('m', euiTheme)}
+    }
+
+    h3 {
+      ${euiTitle('s', euiTheme)}
+    }
+
+    h4,
+    dt {
+      ${euiTitle('xs', euiTheme)}
+    }
+
+    h5 {
+      ${euiTitle('xxs', euiTheme)}
+    }
+
+    h6 {
+      ${euiTitle('xxxs', euiTheme)}
+      text-transform: uppercase;
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6,
+    dt {
+      color: inherit;
+    }
+
+    pre {
+      white-space: pre-wrap;
+      background: ${euiTheme.colors
+        .lightestShade}; // TODO: $euiCodeBlockBackgroundColor - switch to var once EuiCode is converted
+      color: ${euiTheme.colors
+        .text}; // TODO: $euiCodeBlockColor - switch to var once EuiCode is converted
+    }
+
+    pre,
+    pre code {
+      display: block;
+    }
+
+    code {
+      // TODO: Use euiCodeFont mixin once EuiCode is converted
+      font-family: ${euiTheme.font.familyCode};
+      letter-spacing: normal;
+    }
+
+    > :last-child,
+    .euiTextColor > :last-child {
```

Comment:
This selector was to circumvent the double-nest of divs when `color` was passed to `EuiText`, it's no longer needed 🙌 
```suggestion
  {
```

Reply:
https://github.com/elastic/eui/pull/5895/commits/e9c944fa2bef810685895f418f0c4da7fa441980